### DISCUSSION TITLE
Start Bar Double Hight

### DISCUSSION THREAD

**Discussion by Tularis1** [valinet/ExplorerPatcher#3766] [Q&A]
I could swear that before I upgraded to 24H2 my Start Bar was double height.
I have been through every single Explorer Patcher option and I can not see this option.

Any Ideas?

**Comment by pyrates999** [comment]
do you mean taskbar?  You should have uninstalled EP before upgrading to windows 11 24h2.

Open up EP Properties application, go to taskbar, and set **taskbar style** to **Windows 10 (ExplorerPatcher)**, _not_ **Windows 10**.  We no longer offer more support on the stock Windows 10 taskbar.

And to change the height of the taskbar, right click on it and untick **Lock the taskbar**.  Then you can adjust the height of it.
